### ISSUE TITLE
[UI Test] - Error displayed on screen when reading order mocks

### ISSUE THREAD

**Issue by jostnes** [woocommerce/woocommerce-android#7317]
**Describe the bug**
From https://github.com/woocommerce/woocommerce-android/pull/7302#pullrequestreview-1090873640

When the test opens up the orders screen for the first time, there is an error notice displayed on the screen. Currently, it doesn't block the test from running but unsure if this will cause flakiness in the long run. Creating this issue to investigate the reason for the error and fix it.

**Screenshots**
![image](https://user-images.githubusercontent.com/17252150/187827106-03636f21-b7a2-4887-86e6-218157b10913.png)

**Expected behavior**
Error shouldn't be displayed on screen when running test

**Comment by pachlava** [comment]
Thanks for posting this issue @jostnes !

It was consistently reproducible fore me while I was on a certain branch (for another task) that I've created a few days ago. And I made a fix in that branch. Then, in order to have a beautiful fixing branch name 😅, I recreated a new branch from this issue - and it was never reproducible for me again 😕 

I'm leaving here the way I "fixed" it:
- When opening Orders list, for the orders that did not have `completed` status (e.g. for order `2789` which is `on-hold`, and for `2788` which is `processing`), the app tried to change the order status to `completed` for some reason...
- After checking recent changes in mocks, I noticed that these lines were added (https://github.com/woocommerce/woocommerce-android/pull/7296/commits/84e8b43a62eda445dcbb79dc2078c8075d0e6349) to numerous orders mocks:

```
...
"payment_url": "",
"is_editable": false,
...
```

After some thinking I had an assumption that `"is_editable": false` is the cause of the issue: possibly the app saw that the order should not be editable (should be in some final state, with no further transitions), and tried to force the status change to `completed` for orders that had `"is_editable": false`, but did not have status `completed`.

Changing the value to `true` for all non-completed orders solved the issue for me locally.

But since it's no longer reproducible, I'm just leaving this comment, without any action.
